Src/ILGPU/Backends/PointerViews/LowerPointerViews.cs

```diff
@@ -81,10 +86,19 @@ public PointerViewLowering(Method.Builder builder)
             TypeLowering<ViewType> _,
             GetViewLength value)
         {
-            var length = context.Builder.CreateGetField(
+            var builder = context.Builder;
+            var length = builder.CreateGetField(
                 value.Location,
                 value.View,
                 new FieldSpan(1));
+
+            // Convert to a 32bit length value
+            if (value.Is32Bit)
```

Comment:
ISSUE: Should this be the inverse?

```suggestion
            if (!value.Is32Bit)
```

Reply:
I have to admit that this property name is slightly misleading. `Is32Bit` returns true if the type of the `GetViewLength` IR node is essentially a 32-bit value. In this case we need to convert the 64-bit length information to a 32-bit int value. However, we should find a better name 😄